python/lsst/ip/diffim/dcrModel.py

```diff
@@ -126,15 +126,21 @@ def fromDataRef(cls, dataRef, datasetType="dcrCoadd", numSubfilters=None, **kwar
         modelImages = []
         filterInfo = None
         psf = None
+        mask = None
+        variance = None
         for subfilter in range(numSubfilters):
             dcrCoadd = dataRef.get(datasetType, subfilter=subfilter,
                                    numSubfilters=numSubfilters, **kwargs)
             if filterInfo is None:
                 filterInfo = dcrCoadd.getFilter()
             if psf is None:
                 psf = dcrCoadd.getPsf()
-            modelImages.append(dcrCoadd.maskedImage)
-        return cls(modelImages, filterInfo, psf)
+            if mask is None:
+                mask = dcrCoadd.mask
+            if variance is None:
+                variance = dcrCoadd.mask
```

Comment:
I'm confused by these lines: the filter, psf, mask and variance always get set to the first subfilter?

Reply:
Yes, because those must all be the same for every subfilter. But, I notice there's a pretty bad typo here, with the variance being set to equal the mask. I'll fix that.